Minio.Functional.Tests/MintLogger.cs

```diff
@@ -72,12 +72,11 @@ internal class MintLogger
 
       // actual low level exception/error thrown by the program
       public string error {get; private set;}
-
-      public MintLogger(string function, string description,TestStatus status,System.TimeSpan duration,string alert = "",string message="", string error="", Dictionary<string,string> args=null)
+      public MintLogger(string testName,string function, string description,TestStatus status,System.TimeSpan duration, string alert = null,string message=null, string error=null, Dictionary<string,string> args=null)
       {
         this.function = function;
         this.duration = (int)duration.TotalMilliseconds;
-
+        this.name = this.name + ":" + testName;
```

Comment:
a space after the ":" will be more readable, if we have to look at it.

Reply:
sure.